schemast/printer_test.go

```diff
@@ -139,3 +139,9 @@ func (p *printTest) getType(name string) *gen.Type {
 	}
 	return nil
 }
+
+func (p *printTest) getContents(fname string) string {
```

Comment:
```suggestion
func (p *printTest) contents(fname string) string {
```

The `get` prefix is not needed here. It's fine for the `getType` above because `type` is a keyword.